### ISSUE TITLE
Add npm/bower install as part of the setup install process.

### ISSUE THREAD

**Issue by stefanoborini** [simphony/simphony-remote#486]

**Comment by martinRenou** [comment]
I disagree with you on that point, there is two separate actions: `npm install` which install the dependencies and `npm run build` which create our bundle of Javascript.
That's exactly the same in python, you have `pip install -r requirements` and `python setup.py install`.

**Comment by stefanoborini** [comment]
It has to do with pip automatic installation when you access from pypi.

**Comment by stefanoborini** [comment]
Check what jupyterhub does when it's time to deploy the javascript part (if it's compiled/packed)
